backend/experiment/actions/utils.py

```diff
@@ -96,3 +96,12 @@ def get_last_n_turnpoints(session, num_turnpoints):
 
 def randomize_playhead(min_jitter, max_jitter, silence_time, continuation_correctness):
     return silence_time + (random.uniform(min_jitter, max_jitter) if not continuation_correctness else 0)
+
+
+def is_audio_file(section):
+    from section.validators import audio_file_validator
+    try:
+        audio_file_validator(str(section.filename))
+        return True
+    except:
+        return False
```

Comment:
I would expect a (Django) validator to be used in a form and not as a utility function that is used in the Playback class. You could also make it as easy as this and you wouldn't need to use a try-catch.

```py
audio_extensions = ['.wav', '.mp3', '.aac', '.ogg', '.flac']

def is_audio_file(section):
    filename = section.filename
    return any(filename.lower().endswith(ext) for ext in audio_extensions)
```

Reply:
I thought to reuse the validator as it's already there, but you are right, it's a bit weird, and also doesn't number the number of lines added. I also should still be writing a unit test for the part of the playback that determines the `play_method`.